<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propAnnotationTriple</class>
 <widget class="QWidget" name="propAnnotationTriple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>252</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="annotationTriple">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>194</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>AnnotationTriple</string>
   </property>
   <widget class="QDoubleSpinBox" name="ctrZ">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ctrX">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ctrY">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 127, 0);</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="cxLabel">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cyLabel">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="czLabel">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="normZ">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="normY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 127, 0);</string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="nxLabel">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="normX">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="nzLabel">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nyLabel">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLabel" name="nLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Front XYZ: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="FX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-4.8666</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>88.5959</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FZ">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>40.5434</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Left XYZ: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LZ">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-55.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Right XYZ: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="RY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RZ">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>55.0</string>
    </property>
   </widget>
   <zorder>nyLabel</zorder>
   <zorder>nzLabel</zorder>
   <zorder>cxLabel</zorder>
   <zorder>nxLabel</zorder>
   <zorder>cyLabel</zorder>
   <zorder>czLabel</zorder>
   <zorder>ctrZ</zorder>
   <zorder>ctrX</zorder>
   <zorder>ctrY</zorder>
   <zorder>normZ</zorder>
   <zorder>normY</zorder>
   <zorder>normX</zorder>
   <zorder>cLabel</zorder>
   <zorder>nLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>FX</zorder>
   <zorder>FY</zorder>
   <zorder>FZ</zorder>
   <zorder>label_3</zorder>
   <zorder>LY</zorder>
   <zorder>LZ</zorder>
   <zorder>LX</zorder>
   <zorder>label_4</zorder>
   <zorder>RY</zorder>
   <zorder>RZ</zorder>
   <zorder>RX</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ctrX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedCtrX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedCtrY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedCtrZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedNormX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedNormY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationTriple</receiver>
   <slot>changedNormZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedCtrX(double)</slot>
  <slot>changedCtrY(double)</slot>
  <slot>changedCtrZ(double)</slot>
  <slot>changedNormX(double)</slot>
  <slot>changedNormY(double)</slot>
  <slot>changedNormZ(double)</slot>
  <slot>changedSize(int)</slot>
 </slots>
</ui>
